File: app/src/main/java/com/example/administrator/jipinshop/activity/follow/user/UserView.java
package com.example.administrator.jipinshop.activity.follow.user;

import com.example.administrator.jipinshop.bean.SuccessBean;
import com.example.administrator.jipinshop.bean.UserPageBean;

/**
 * @author 莫小婷
 * @create 2018/11/17
 * @Describe
 */
public interface UserView {
    void onSuccess(UserPageBean userPageBean);
    void onFile(String error);

    void ConcerDelSuccess(SuccessBean successBean);
    void ConcerDelFaile(String error);

    void concerInsSuccess(SuccessBean successBean);
}

File: app/src/main/kotlin/com/example/administrator/jipinshop/activity/minekt/userkt/UserView.kt
package com.example.administrator.jipinshop.activity.minekt.userkt

import com.example.administrator.jipinshop.bean.UserInfoBean

/**
 * @author 莫小婷
 * @create 2019/8/19
 * @Describe
 */
interface UserView {
    fun onSuccess( bean : UserInfoBean)
    fun onFile( error : String?)

    fun onAttent()
    fun onCancleAttent()
}